src/main/java/uk/gov/justice/digital/delius/config/SwaggerConfig.java

```diff
@@ -61,11 +86,43 @@ private Contact contactInfo() {
     private ApiInfo apiInfo() {
         return new ApiInfo(
                 "Community API Documentation",
-                "REST service for accessing community probation information\nThis service serves two sets of APIs, those secured by OAuth2 (and marked as Secure) and those internal to the Probation New Tech Application which are not accessible to any other applications",
+                "<h2>REST service for accessing community probation information</h2>" +
+                        "<p>This service provides endpoints for accessing data primary sourced from National Delius about people that are of interest to HM Probation Service.</p>" +
+                        "<p>There is cross-over with the <b>prison-api</b> though suspects on remand will not be surfaced by this API unless that have previously been on probation.</p>" +
+                        "<div>" +
+                        "This service is secured by <b>OAuth2</b> with tokens supplied by HMPPS Auth. Most read-only endpoints require the <b>ROLE_COMMUNITY</b> to access, but check each endpoint where this differs." +
+                        "<p>This service can be accessed in a number environments. For each environment a different set of OAuth2 credentials from HMPPS Auth are required</p>" +
+                        "<ul>" +
+                        "<li>Test/Development: <b>https://community-api-secure.test.delius.probation.hmpps.dsd.io</b></li>" +
+                        "<li>Pre-production: <b>https://community-api-secure.pre-prod.delius.probation.hmpps.dsd.io</b></li>" +
+                        "<li>Production: <b>https://community-api-secure.probation.service.justice.gov.uk</b></li>" +
+                        "</ul>" +
```

Comment:
Couldn't see another way of specifying this URLs, there is even a new` Docklet.servers()` method  that has been added but to doesn't produce any swagger